Source/DafnyCore/AST/MemberDecls.cs

```diff
@@ -128,7 +128,7 @@ public Field(RangeToken rangeToken, Name name, bool hasStaticKeyword, bool isGho
 
   public bool SetIndent(int indentBefore, TokenNewIndentCollector formatter) {
     formatter.SetOpeningIndentedRegion(StartToken, indentBefore);
-    formatter.SetClosingIndentedRegion(EndToken, indentBefore);
+    formatter.SetIndentations(EndToken, below: indentBefore);
```

Comment:
Before, it would set the indent of the last token to (above:Indent + 2, inline:Indent, below:Indent), which works for sets and sequences, but not for constants.
Now I only set the indentation below that last token, which is the only thing really needed.